datadog_checks_dev/datadog_checks/dev/__about__.py

```diff
@@ -1,4 +1,4 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-__version__ = '0.39.0'
+__version__ = '1.0.0'
```

Comment:
/!\ The version was changed in the CHANGELOG but not here. As a result a `v1.0.0` release was made to PyPI: https://pypi.org/project/datadog-checks-dev/ (and 0.40.0 doesn't exist).